### PR TITLE
Lm employee details

### PR DESCRIPTION
**Pull request by LukeAMiller**
# Description
Details page for Employees

## Related Ticket(s)
3

## Steps to Test

- Play VS with green play button
-On the provided webpage click employees link
-on the list of  employees click a details link for an employee
Tadaa

# Checklist:
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new errors

### CODE REVIEW

File: BangazonWorkforce/Views/Employee/Index.cshtml [hunk 1]
```diff
@@ -1,15 +1,35 @@
 ﻿@model IEnumerable<BangazonWorkforce.Models.Employee>
+<<<<<<< HEAD
```
Comment (jordan-castelloe) [inline BangazonWorkforce/Views/Employee/Index.cshtml]: You have some merge conflicts here-- please clean these up before anyone reviews 👍

File: SQLQuery4.sql [hunk 2]
```diff
@@ -0,0 +1,2 @@
+﻿SELECT * FROM Department
```
Comment (jordan-castelloe) [inline SQLQuery4.sql:1]: Did you mean to push up this file?

File: SQLQuery5.sql [hunk 3]
```diff
@@ -0,0 +1,5 @@
+﻿SELECT  Employee.Id AS 'Id', FirstName, 
+LastName, Department.Name AS 'DeptName', isSupervisor ,  Computer.Make, Computer.Manufacturer
```
Comment (jordan-castelloe) [inline SQLQuery5.sql:2]: Did you mean to push up this file as well? If you have  SQL queries that you're testing, you don't need to save them and commit them. If they're queries that other people need, you should save them with clear naming conventions and describe what they do in your PR

**Review by jordan-castelloe — CHANGES_REQUESTED** [review]
- Please leave more thorough testing instructions. What should the user see when they see the details? Give us an example-- like if the user clicks on the details for Bob Ross, which computer is his current computer? What training programs is he in?

**Comment by AllxnCook** [comment]
works as stated.
more comments.
more detailed instructions on how to run the program.

**Comment by LanceP51** [comment]
WOW, great functionality with the crappiest PR ever!

**Pull request merged**